test/test_hypergeometric_dist.cpp

```diff
@@ -493,6 +493,19 @@ void test_spots(RealType /*T*/, const char* type_name)
    BOOST_MATH_CHECK_THROW(d = boost::math::hypergeometric_distribution<RealType>(40, 501, 500), std::domain_error);
 }
 
+// BOOST_AUTO_TEST_CASE(test_non_integer_k)
+// {
+//    boost::math::hypergeometric_distribution<double> dist(30, 10, 15);  // N=30, r=10, n=15
+
+//    BOOST_CHECK_CLOSE(
+//       pdf(dist, 4.5),
+//       pdf(dist, 4.0) * 0.5 + pdf(dist, 5.0) * 0.5,
+//       0.1);  // Interpolación lineal
+
+//    // Valores fuera del soporte deben dar 0
```

Comment:
If you leave this comments in can you please translate them to English.

Reply:
Sorry, that was commited by mistake, reverting...